src/assets/wise5/authoringTool/node/nodeAuthoring.html

```diff
@@ -26,39 +26,39 @@
       </md-tooltip>
     </md-button>
     <md-button id='addComponentButton'
-        ng-if='$ctrl.showStepButtons && !projectController.isGroupNode($ctrl.node.id)'
+        ng-if='$ctrl.showStepButtons && !$ctrl.node.isGroup()'
```

Comment:
Maybe change all isGroup() calls in this file to reference a boolean variable instead too.

Reply:
Good suggestion. I made the change.